corecito.py

```diff
@@ -129,7 +129,10 @@ async def main():
         await asyncio.sleep(1)
         break
       else:
-        logger.info("Exception occurred -> '{}'. Waiting for next iteration... ({} seconds)\n\n\n".format(e, config['seconds_between_iterations']))
+        error_message = "Exception occurred -> '{}'. Waiting for next iteration... ({} seconds)\n\n\n".format(e, config['seconds_between_iterations'])
+        if config['telegram_notify_errors']:
+          telegram.send(error_message)
+        logger.info(error_message)
```

Comment:
I think it's a great idea. As for the implementation, instead of adding more logging code  to this main file we are gradually moving all the logs to a separate file `logger.py`.
Could you please refactor this error log by doing something similar to `logger.logBuyMissing`?

Reply:
Sure! ☺️ 
I took advantage of this PR to remove a couple of withespaces that were hurting my eyes 😛

Reply:
Nice, we could add a lint tool to clean-up white-spaces and add a consistent code style in the project.
But I have no idea how those work in Python because I don't use an IDE for python, just a text editor.